README.md

```diff
@@ -1 +1,81 @@
 # Immutable Web Apps
+
+## Introduction
+
+_Immutable Web Applications_ is a framework-agnostic methodology for building static [single-page application](https://en.wikipedia.org/wiki/Single-page_application) that:
```

Comment:
Do you want to keep the methodology and principals focused on just building, or does it make sense to extend to deployment?  I suppose there are many deployment options and whatever you choose for deploying index and assets, it doesn't fundamentally change anything.  I do think for the ideas presented to be more fully understood it will be important to show one or more deployment examples.

Reply:
Yeah, frankly, I feel like the deployment aspect is the more important part. Are you suggesting that I change the line to:

> _Immutable Web Applications_ is a framework-agnostic methodology for building **and deploying** static single-page applications

Reply:
Just to respond to your comment more generally, yes, I think it would be great to create a couple of very specific deployment examples. Using AWS certainly, and then maybe another couple popular platforms/tech stacks.

Reply:
Yep, that is all I was suggesting, adding the 'and deploying' 👍